sourcemash/frontend/templates/forgot-modal.tpl

```diff
@@ -0,0 +1,24 @@
+<div class="modal-content">
+  <h4>Send password reset instructions</h4>
+  <p class="divider"><p>
+  <p class="flow-text">An email will be sent to your account with further instructions upon submission.<p>
+  <div class="row">
+    <form id="forgot-form">
+      <div class="input-field">
+        <input class="validate" type="email" name="email">
+        <label for="forgot-email" class="">Email Address</label>
+        <div class="errors" id="forgot-errors"></div>
+      </div>
+      <div class="row">
+        <div class="input-field col s12">
+          <button type="submit" class="btn waves-effect waves-light">Recover Password</button>
+        </div>
+      </div>
+    </form>
+  </div>
+</div>
+<p class="divider"><p>
+<div class="modal-footer">
+  <a href="#" class="forgot-close waves-effect waves-green btn-flat modal-action modal-close">I remember!</a>
+  <a href="#" class="waves-effect waves-green btn-flat modal-action modal-close">I'll pass...</a>
```

Comment:
For functionality, no? For consistency and based on the fact that you had kinda mentioned you like the option for a "cancel" button, yes. It does no harm and it's on every other modal, which is the only reason that I included it.